components/script/dom/performanceresourcetiming.rs

```diff
@@ -194,4 +194,9 @@ impl PerformanceResourceTimingMethods for PerformanceResourceTiming {
     fn FetchStart(&self) -> DOMHighResTimeStamp {
         Finite::wrap(self.fetch_start)
     }
+
+    // https://w3c.github.io/resource-timing/#dom-performanceresourcetiming-connectend
+    fn ConnectEnd(&self) -> DOMHighResTimeStamp {
+        Finite::wrap(self.connect_end)
```

Comment:
You will need to update PerformanceResourceTiming::from_resource_timing as well to initialize this member.

Reply:
Fixed in c4bc87709cd8b4f8f510960d996c33b7a29f2c62.